Mathlib/Tactic/Recover.lean

```diff
@@ -5,11 +5,20 @@ Authors: Gabriel Ebner, Siddhartha Gadgil, Jannis Limperg
 -/
 import Lean
 
-open Lean (HashSet)
-open Lean Meta Elab Tactic
+/-!
+# The `recover` tactic modifier
+
+This defines the `recover` tactic modifier, which can be used to debug cases where goals
+are not closed correctly. `recover tacs` for a tactic (or tactic sequence) `tacs`
+applies the tactics and then adds goals
+that are not closed, starting from the original goal.
```

Comment:
Please double-check this phrasing; I was not fully sure how to parse the original comment.